test/test_iperf.ml

```diff
@@ -194,42 +194,42 @@ let test_tcp_iperf_two_stacks_basic amt timeout () =
   let module Test = Test_iperf (Vnetif_backends.Basic) in
   Test.default_network () >>= fun { backend; Test.client; Test.server } ->
   Test.V.record_pcap backend
-    (Printf.sprintf "tests/pcap/tcp_iperf_two_stacks_basic_%d.pcap" amt)
+    (Printf.sprintf "tcp_iperf_two_stacks_basic_%d.pcap" amt)
     (Test.tcp_iperf ~server ~client amt timeout)
 
 let test_tcp_iperf_two_stacks_mtu amt timeout () =
   let module Test = Test_iperf (Vnetif_backends.Mtu_enforced) in
   Test.default_network () >>= fun { backend; Test.client; Test.server } ->
   Test.V.record_pcap backend
-    (Printf.sprintf "tests/pcap/tcp_iperf_two_stacks_mtu_%d.pcap" amt)
+    (Printf.sprintf "tcp_iperf_two_stacks_mtu_%d.pcap" amt)
```

Comment:
I think this will enable pcap recording by default in CI as the dir will now exist. If CI becomes slow it may be because of the large files (unless the pcap recording has been fixed to record less data since then)